engines/group/app/controllers/group/api/v1/memberships_controller.rb

```diff
@@ -9,7 +9,8 @@ class MembershipsController < ApplicationController
         # GET /api/v1/memberships
         #
         def index
-          memberships = Membership.where(user_id: current_user.id)
+          group_ids = current_user.groups.map(&:id)
+          memberships = Membership.where(group_id: group_ids)
```

Comment:
```ruby
> Group::Membership.where(group_id: user.group_ids)
   (1.4ms)  SELECT "groups".id FROM "groups" INNER JOIN "memberships" ON "groups"."id" = "memberships"."group_id" WHERE "memberships"."user_id" = $1  [["user_id", 1]]
  Group::Membership Load (0.3ms)  SELECT "memberships".* FROM "memberships" WHERE "memberships"."group_id" = 1
```

2 queries, but I still don't like the `JOIN` on the first. Maybe we can find something better than this.